### PR TITLE
Release v15

### PR DESCRIPTION
**Pull request by himanshuchawla009**

### CODE REVIEW

File: packages/core/src/core.ts [hunk 1]
```diff
@@ -205,21 +213,36 @@ class ThresholdKey implements ITKey {
...
-    withShare?: ShareStore;
-    importKey?: BN;
-    neverInitializeNewKey?: boolean;
-    transitionMetadata?: Metadata;
-    previouslyFetchedCloudMetadata?: Metadata;
-    previousLocalMetadataTransitions?: LocalMetadataTransitions;
-    delete1OutOf1?: boolean;
-  }): Promise<KeyDetails> {
+  getSecp256k1Key(): BN {
+    if (typeof this.privKey !== "undefined") {
```
Comment (chaitanyapotti) [inline packages/core/src/core.ts:217]: can make these getters

File: packages/tss/src/tss.ts [hunk 2]
```diff
@@ -0,0 +1,811 @@
...
+          return new BN(importKey);
+        } else if (this._tssKeyType === KeyType.ed25519) {
+          // Store seed in metadata.
+          const domainKey = getEd25519SeedStoreDomainKey(this.tssTag || TSS_TAG_DEFAULT);
+          const result = this.metadata.getGeneralStoreDomain(domainKey) as Record<string, unknown>;
+          if (result) {
+            throw new Error("Seed already exists");
+          }
+
+          const { scalar } = getEd25519ExtendedPublicKey(importKey);
```
Comment (ieow) [inline packages/tss/src/tss.ts:334]: function name should be renamed to getEd25519KeyPairFromSeed ?

**Review by ieow — APPROVED** [review]
lgtm

File: packages/core/src/core.ts (hunk 1 above)
Comment (himanshuchawla009) [reply packages/core/src/core.ts]: done

File: packages/tss/src/tss.ts (hunk 2 above)
Comment (himanshuchawla009) [reply packages/tss/src/tss.ts]: done

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**